### ISSUE TITLE
Found Inconsistencies - Still Active?

### ISSUE THREAD

**Issue by ioneyed** [skeleton-framework/skeleton-framework#52]
I was wondering if this project is still active because I found some inconsistencies with versioning and what your marketing site is using.

1) your marketing site is using skeleton.css version 1.0.4 yet your npm packages start with 1.0.7
2) I attempted to recreate your base site for a "bone" that I am working on however the differences in 1.0.10/9 make grid vastly different in your marketing site. (basically eliminates the gutters between the columns).

If this project isn't active, I thank you for all your work and will fork/create my own based on this work.

**Comment by rafgugi** [comment]
how do we know if this project isnt active?

**Comment by ioneyed** [comment]
No commits in 4 months, their main site is using out dated version of their own framework.. General signs of abandonment or staleness.

**Comment by enlore** [comment]
A bit stale, yeah. Abandoned, no. We live! And we welcome PRs if you'd like to help tighten things up. I'm afraid I'm guilty of using the framework more or less as it stands (with a few tweaks) and not circling back here to publish updates.

Knowing that people are using it is motivating, though. Let me know if you have any questions pursuant to crafting your Bone and I'll put time in this week to update the site, sync up the versions.

Thanks for pinging.

**Comment by GusBeare** [comment]
I like what I see and I'd like to use it but I got the css from getskeleton.com and I see this at the top of skeleton.css:

/*
* Skeleton V2.0.4
* Copyright 2014, Dave Gamache
* www.getskeleton.com
* Free to use under the MIT license.
* http://www.opensource.org/licenses/mit-license.php
* 12/29/2014
*/

I'm confused about which version I should be using?

**Comment by enlore** [comment]
The newer version. The version from getskeleton.com is the older, unmaintained version.

**Comment by GusBeare** [comment]
ok, thanks

**Comment by rafgugi** [comment]
btw I have an idea to rebuild skeleton using sass (or scss), but I am still new to it and dont know where to start. but I know its good because most frontend library use it

**Comment by GusBeare** [comment]
yes, great, good idea. Thanks for keeping this alive. Bootstrap is too big for some of the simpler sites I do.

**Comment by mef** [comment]
> I attempted to recreate your base site for a "bone" that I am working on however the differences in 1.0.10/9 make grid vastly different in your marketing site. (basically eliminates the gutters between the columns).

You may want to use `dist/index.html`, which is more up-to-date than the martketing site, and includes newer features of skeleton framework (such as cards).

**Comment by mvdkleijn** [comment]
Mayhaps someone should update the marketing site?

**Comment by andrewbash** [comment]
Seeing as I've used this framework for just about every project for the past several years I would be happy to update the marketing site... I just can't find where to contribute to it... I don't see a github pages branch, or anywhere in the repo the github page would be pulling from.

Any advice?

**Comment by enlore** [comment]
Would love to have the help. Let me circle back to this in the next couple days and I’ll get you set up.

**Comment by xeruf** [comment]
@enlore still actively using it?

See also #6
